legal-api/migrations/versions/e6ff12b18bef_.py

```diff
@@ -0,0 +1,44 @@
+"""empty message
+
+Revision ID: e6ff12b18bef
+Revises: 026f42f4f654
+Create Date: 2021-12-01 12:32:41.850361
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'e6ff12b18bef'
+down_revision = '026f42f4f654'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('resolutions', sa.Column('resolution', sa.String(length=4096), nullable=True))
+    op.add_column('resolutions', sa.Column('signing_date', sa.Date(), nullable=True))
+    op.add_column('resolutions', sa.Column('signing_party_id', sa.Integer(), nullable=True))
+    op.add_column('resolutions', sa.Column('sub_type', sa.String(length=20), nullable=True))
+    op.add_column('resolutions_version', sa.Column('resolution', sa.String(length=4096), nullable=True))
```

Comment:
schema and UI validation has max size at 1000 for validation but this length is probably a good call.  Gives us some wiggle room if we ever need to up it.  👍